cmd/server.go

```diff
@@ -123,10 +127,13 @@ const (
 	DefaultDataDir          = "~/.atlantis"
 	DefaultGHHostname       = "github.com"
 	DefaultGitlabHostname   = "gitlab.com"
+	DefaultLockingDBType    = "boltdb"
 	DefaultLogLevel         = "info"
 	DefaultParallelPoolSize = 15
 	DefaultStatsNamespace   = "atlantis"
 	DefaultPort             = 4141
+	DefaultRedisHost        = "redis"
```

Comment:
Good catch, I think I had that in there while testing. I removed it as a default since, ideally, if you're using this, you should know your Redis hostname. Removed.